Signal/src/ViewControllers/ContactViewController.swift

```diff
@@ -417,6 +420,14 @@ class ContactViewController: OWSViewController, CNContactViewControllerDelegate
         circleView.autoPinEdge(toSuperviewEdge: .top)
         circleView.autoHCenterInSuperview()
 
+        guard let image = UIImage(named: imageName) else {
```

Comment:
Reminder that you can better capture this pattern with declarative resources - and have the IDE make sure you didn't typo it.

Reply:
I'm still thinking this over.